app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -24,14 +38,37 @@ class MainActivity : ComponentActivity() {
 
 @Composable
 fun PlotSurface() {
+    var xPercent: Float by remember { mutableStateOf(0.5f) }
+    var yPercent: Float by remember { mutableStateOf(0.5f) }
     Surface(
         modifier = Modifier.fillMaxSize(),
         color = Color.White
     ) {
-        // TODO Build out the plot surface
-        //  This should include a Column composable that
-        //  includes a Map, and two MapSlider composables
-        //  (one slider for each axis).
+        Column (
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center
+        ) {
+            Box(Modifier.background(Color.Blue).size(300.dp)){
+                Map(xPercent,yPercent)
+            }
+
+
+            createHorizontalSlider(value = xPercent,
+                changeValue = { value ->
+                    xPercent = value
+                }
+            )
+            createVerticalSlider(value = yPercent,
+                changeValue = { value ->
+                    yPercent = value
+                    //Log.i("y-tag", (yPercent * 300 - 18).toString())
```

Comment:
nit pick here, but you could remove this commented log.  Nice job debugging tho!